src/shared/dropdown-wrapper/element-with-dropdown.tsx

```diff
@@ -59,7 +61,8 @@ export const ElementWithDropdown = ({
     onDismiss,
     renderElement,
     renderDropdown,
-    zIndex = 50,
+    customZIndex,
+    defaultZIndex = 50,
```

Comment:
`defaultZIndex`, this can be define as a const in the file right? 

If they ever need their own zIndex, can just use the `customZIndex`. 

And if default can just declare in the file, custom can be reverted to zIndex but optional